src/grids.jl

```diff
@@ -0,0 +1,96 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    geotiff(Datumₛ, Datumₜ)
+
+GeoTIFF file used in transforms that convert source `Datumₛ` to target `Datumₜ`.
+"""
+function geotiff end
+
+const INTERPOLATION = IdDict()
```

Comment:
needs comment above explaining the purpose of this global const dict for future readers